packages/st2mistral/Makefile

```diff
@@ -1,8 +1,14 @@
+# Mind that that some of the vars are passed as ENV!
 WHEELDIR ?= /tmp/wheelhouse
-ST2_COMPONENT := $(notdir $(CURDIR))
+COMPONENT := st2mistral
 MISTRAL_RELEASE ?= 1
-# hardcode it for now since st2mistral doesn't have it's version
-MISTRAL_VERSION ?= 1.2.0
+MISTRAL_VERSION ?= $(shell python -c "from pbr import version; print version.VersionInfo('mistral').version_string(),")
+
+# We use special additional requirements in our mistral bundle!
+define INJECT_DEPS
+gunicorn
+-e git+https://github.com/StackStorm/st2mistral.git@$(GITREV)#egg=st2mistral
```

Comment:
It appears we should also include python-mistralclient which gets the mistral CLI as well. python-mistralclient also follows the same branching mechanism as st2mistral. See https://stackstorm.slack.com/archives/packages/p1454525032003033 for why.